semtypes/src/main/java/io/ballerina/types/PredefinedType.java

```diff
@@ -106,8 +106,8 @@ public final class PredefinedType {
                     | (1 << BasicTypeCode.BT_STRING.code));
 
     public static final SemType IMPLEMENTED_TYPES =
-            union(OBJECT, union(FUNCTION, union(SIMPLE_OR_STRING, union(XML, union(HANDLE, union(REGEXP, union(FUTURE,
-                    union(ERROR, union(STREAM, union(TYPEDESC, union(LIST, MAPPING)))))))))));
+            union(FUNCTION, union(SIMPLE_OR_STRING, union(XML, union(HANDLE, union(REGEXP, union(FUTURE,
```

Comment:
@heshanpadmasiri it was intentional. OBJECT type should be added after #43010 since the object type is not implemented yet in the nutcracker branch. I have test failures with the `OBJECT` here.